test/MenuItem/SelectableItemRendererTest.php

```diff
@@ -84,16 +84,14 @@ public function testRenderDisabled() : void
         $menuStyle->setWidth(35);
         $style = (new SelectableStyle())->setItemExtra('[DONE]');
 
-        $item = new SelectableItem('SOME TEXT', function () {
+        $item = new SelectableItem($testString = 'SOME TEXT', function () {
         });
         $item->setStyle($style);
         $item->showItemExtra();
 
         self::assertEquals(
-            [
-                "\033[2m● SOME TEXT\033[22m              [DONE]",
-            ],
-            $renderer->render($menuStyle, $item, true, true)
+            escapeshellcmd("\033[2m\033[97m● " . $testString . "\033[37m\033[22m              [DONE]"),
```

Comment:
Whats the `escapeshellcmd` for?

Reply:
If this test ended up failing (for whatever reason), when throwing the error back to the console, PHPUnit would inadvertently break the styling output of its own process by attempting to display the test string. I think it was trying to truncate it, if I recall correctly.

Comparing escaped shell commands solved it in this case. I did verify using Xdebug and earlier tests that a shorter string without `escapeshellcmd` still passes the test.

Reply:
Sounds good, maybe in the future we can provide some utils for the tests using this method.